docs/adr/0024-upgrade-templating-toolset-of-auth-server-emails.md

```diff
@@ -0,0 +1,177 @@
+# Upgrade the templating toolset of Auth Server emails
+
+- Status: accepted
+- Deciders: Yogita Bhatia, Lauren Zugai, Jody Heavener, Vijay Budhram, Ben Bangert, Les Orchard, Danny Coates
+- Date: 2021-06-17
+
+Technical Story: [#4627](https://github.com/mozilla/fxa/issues/4627) / [#8600](https://github.com/mozilla/fxa/issues/8600)
+
+## Context and Problem Statement
+
+[Mustache](https://www.npmjs.com/package/mustache), used in `fxa-auth-server` email templates, can accommodate very limited logic in its templates which has been a pain point for Firefox Accounts engineers and discussion around using a different templating system [began here](https://github.com/mozilla/fxa/issues/4627). While converting our emails to a more modern templating solution, there is an opportunity to evaluate what stack would be the most ideal for FxA emails beyond a proposed templating solution. This includes evaluating our CSS options, improving how we can preview emails in various states, and our localization tools, and the best approach for landing new templates in production.
+
+## Decisions and Considered Options
+
+### Templating and Styling
+
+- Option A - Continue to use Mustache and ad-hoc inline styles
+- Option B - Use React server-side to generate static HTML templates with TailwindCSS
+- Option C - Use EJS and MJML, using CSS options offered by MJML
+
+### Email previewing
+
+- Option A - Continue to use the `write-emails` command
+- Option B - Use Storybook
+- Option C - Use Mailtrap
+
+--
```

Comment:
Maybe should be `---`? Missing a `-` if this should be a horizontal rule

Reply:
Ohh, this was `--` because in HackMD, where the collab for this ADR began, it sticks an emdash looking thing here, which looked better to me than the horizontal line. I'm not sure that's "appropriate" here anyway, and I like the horizontal line rather than nothing, so agreed, let's change to `---`